docs/book/src/quinn/set-up-connection.md

```diff
@@ -0,0 +1,89 @@
+# Connection Setup
+
+In the [previous chapter](certificate.md) we looked at how to configure a certificate.
+This aspect is omitted in this chapter to prevent duplication. 
+But **remember** that is is required to get your [Endpoint][Endpoint] up and running. 
+This chapter explains how to set up a connection and prepare it for data transfer. 
+
+It all starts with the [Endpoint][Endpoint] struct, this is the entry of the library. 
+
+## Example
+
+Let's start by defining some constants. 
+
+```rust
+static SERVER_NAME: &str = "localhost";
+
+fn client_addr() -> SocketAddr {
+    "127.0.0.1:5000".parse::<SocketAddr>().unwrap()
+}
+
+fn server_addr() -> SocketAddr {
+    "127.0.0.1:5001".parse::<SocketAddr>().unwrap()
+}
+```   
+
+For both the server and the client we use the [EndpointBuilder][EndpointBuilder]. 
+The [EndpointBuilder][EndpointBuilder] has a method [bind(address)][bind] with which you link an address to the endpoint. 
+This method initializes a UDP-socket that is used by quinn.
+If you need more control over the socket creation, it is also possible to initialize a quinn endpoint with an existing UDP socket with [with_socket][with_socket]. 
+
+**Server**
+
+Just like a TCP Listener, you have to listen to incoming connections.
+Before you can listen to connections you need to configure the [EndpointBuilder][EndpointBuilder] as a server. 
+Note that the configuration itself does not perform any listening logic, instead use the `Incomming` type returned by [bind()][bind].  
```

Comment:
```suggestion
One your [EndpointBuilder][EndpointBuilder] has been configured, it must be bound to a socket address. The [bind()][bind] used to do this will return an `Endpoint` type that can be used to start new outgoing connections as well as an `Incoming` type used to listen for incoming connections.
```

Reply:
The proposed version contains some errors, I see what your trying to point at. I reworte this to:
```
First, the Quinn endpoint should be bound to a socket. The [bind()][bind] method, which can be used for this, returns an `Endpoint` and an `Incomming` type. The `Endpoint` type can be used to start outgoing connections, and the `Incoming` type can be used to listen for incoming connections.
```